### ISSUE TITLE
Home and End buttons not working

### ISSUE THREAD

**Issue by AndrewSav** [PowerShell/PSReadLine#1027]
Environment data
----------------
```text
PS version: 6.2.0
PSReadline version: 2.0.0-beta3
os: Linux shkaf 4.15.0-55-generic #60-Ubuntu SMP Tue Jul 2 18:22:20 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
PS file version: 6.2.0.0
BufferWidth: 194
BufferHeight: 59
```
Steps to reproduce or exception report
--------------------------------------

- Start Powershell
- Type something, e.g. `abcdefghijklmnopqrstuvwxyz`, do not press enter.
- Try pressing `Home` button. Expected: cursor moves to `a`. Observerd: cursor stays where it was.
- Move cursor with arrows to `a` and press `End` button. Expected: curstor moves to `z`. Observerd: cursor stays where it is.

`Home` and `End` buttons work as expected in both powershell 6 on Windows and in bash on Linux, this is why this behaviour is also expected from Powershell on linux.

Note: this issue was open already as https://github.com/PowerShell/PSReadLine/issues/702, but subsequently closed with the reasoning proved at https://github.com/PowerShell/PSReadLine/issues/702#issuecomment-529529428. It advises to create a new issue, which this issue is.

**Comment by rkeithhill** [comment]
I brought up a similar issue (happens to me when SSH'ing into Linux machine) in issue https://github.com/PowerShell/PowerShell/issues/8869

**Comment by AndrewSav** [comment]
@rkeithhill well I came here because another issue on https://github.com/PowerShell/PowerShell that was reporing this was closed with the reason "This is a PSReadLine" issue.

FWIW I'm running all machines on a vSphere host and access them with SSH, so it sounds like this is the same issue as yours. 

Just tryied to loging to one of the VMs via vSphere console and `Home` and `End` are working there.

Okay, I'm going to close this for now here, as I beleive your expert opinion that this is a Powershell issue not PSReadLine.
